dataprofiler/profilers/float_column_profile.py

```diff
@@ -131,11 +142,9 @@ def _is_each_row_float(cls, df_series):
         :return: is_float_col
         :rtype: list
         """
-        len_df = len(df_series)
-        if len_df == 0:
-            return list()
-
-        return [NumericStatsMixin.is_float(x) for x in df_series]
+        if len(df_series) == 0: return list()
+        query = r'^[\s]*[+-]?[\d]+[\.]*[\d]*[ ]?[*eE]*[ ]?[+-]?[\d]*[\s]*$'
```

Comment:
Will need to discuss tomorrow, but I believe there are a few issues with this regex.

There should only be 1 `.`, values such as `.52` I believe work for python, bit would fail this regex.

This is going to take some studying. Will address further in the morning.

Reply:
Since this is getting changed to regex, we need a test for a variety of what we expect for floats.

Reply:
Was outdated anyway, tried it. Was slower. See comment below. Going to dismiss.